bramkouwen/Neatly-In-Motion
# Updating Contact Page

## What does this PR do?
* It updates contact.html 

## How do you check this PR?
* Pull down the branch
* Check the added content to contact.html  

## Screenshots 
None.
